src/widgets/gauge.rs

```diff
@@ -129,6 +134,20 @@ impl<'a> Widget for Gauge<'a> {
     }
 }
 
+fn get_unicode_block<'a>(frac: f64) -> &'a str {
+    match (frac*8.0).round() as u16{ //get how many eighths the fraction is closest to
+        1 => "▏",
```

Comment:
Those graphemes are usually defined in [src/symbols.rs](https://github.com/fdehau/tui-rs/blob/master/src/symbols.rs) in order to potentially be used by other widgets.

Reply:
Now references the symbols in symbols.rs as of the latest commit